### DISCUSSION TITLE
iOS进alist播放视频画中画无法使用

### DISCUSSION THREAD

**Discussion by ethuh** [AlistGo/alist#946] [General]
无论是全屏还是小窗状态下点击画中画都无法弹出来窗口。
小窗口状态下，点击画中画按钮，会有提示消息，如下，然后弹出来的画中画窗口自动缩回小窗口，
![Clipboard 2022年4月15日 23 56](https://user-images.githubusercontent.com/23073985/163593057-8d6f2640-207b-4ac2-8ea5-26d4c5fc052e.jpeg)
全屏状态下，点击画中画或者直接滑到桌面，画中画窗口会自己消失。

nplayer播放的时候，画中画可以弹出，但是过一会儿会缩回，有可能是nplayer网络要求较高的原因。

**Comment by xhofe** [comment]
👉 https://github.com/zhw2590582/ArtPlayer
